src/components/YourCompany/OfficeAddresses.vue

```diff
@@ -95,15 +95,15 @@
         </v-flex>
       </v-layout>
 
-      <v-layout row id="summary-records-address" class="mt-4 mx-0" v-if="!entityFilter(EntityTypes.COOP)">
+      <v-layout row id="summary-records-address" class="mt-4 mx-0" v-if="entityFilter(CorpTypeCd.BENEFIT_COMPANY)">
```

Comment:
I flipped the logic here to be more concise.

Example: what if we have another corp type that, like COOP, should not display this section?